library/ssl_tls.c

```diff
@@ -5956,6 +5970,17 @@ static unsigned char ssl_serialized_session_header[] = {
  * The order is the same as in the definition of the structure, except
  * verify_result is put before peer_cert so that all mandatory fields come
  * together in one block.
+ *
+ * In TLS/DTLS 1.3 the ticket contains the following content:
+ *
+ *  - ticket creation time ( start )
+ *  - ciphersuite ( ciphersuite )
+ *  - ticket lifetime ( ticket_lifetime )
+ *  - ticket-age-add parameter ( ticket_age_add )
+ *  - key length ( key_len )
+ *  - flags ( flags )
+ *  - key ( key )
+ *  - certificate of the peer ( peer_cert )
```

Comment:
I think we should drop the certificate form the ticket in TLS 1.3. The primary reason why we have it in TLS 1.2 is to avoid the triple handshake attack, which AFAIU doesn't exist in TLS 1.3.

Reply:
Removed it.